nx.json

```diff
@@ -152,8 +152,8 @@
         "jest-fixtures"
       ],
       "outputs": [
-        "{projectRoot}/dist/**/fixtures/jasmine",
-        "{projectRoot}/dist/**/fixtures/jest"
+        "{projectRoot}/dist/fixtures/jasmine",
```

Comment:
Do you keep build fixtures, so jest and jasmine seem mixed because of that...

Reply:
I misunderstood your remark, we still build both but they are exported via different entrypoints. 
We should not have reference of jest in jasmine and vice-versa